File: C_Programming/Competitive_Programming/CP_013.c
//////////////////////////////////////////////////////////////////
//
//  File Name    : Display_Even_Factors
//
//  Description  : This program accepts a number from the user
//                 and displays all even factors of the given
//                 number, excluding the number itself.
//
//  Example:
//      Input  : 12
//      Output : 2 4 6
//
//      Input  : 20
//      Output : 2 4 10
//
//////////////////////////////////////////////////////////////////

#include<stdio.h>

void DisplayEvenFactor(int iNo)
{
    int i = 0;

    if(iNo <= 0)
    {
        iNo = -iNo;
    }

    for(i = 1; i < iNo ; i++)
    {
        if((iNo % i == 0) && (i % 2 == 0))
        {
            printf("%d\t", i);
        }
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number : \n");
    scanf("%d", &iValue);

    DisplayEvenFactor(iValue);

    return 0;
}

File: Java_Programming/Competitive_Programming/CP_013.java
//////////////////////////////////////////////////////////////////
//
//  File Name    : Check_Divisibility_By_5_And_11
//
//  Description  : This program accepts a number and
//                 checks whether the given number is
//                 divisible by both 5 and 11.
//
//  Example:
//      Input  : 55
//      Output : Number is Divisible.
//
//      Input  : 50
//      Output : Number is not Divisible.
//
//////////////////////////////////////////////////////////////////

class Logic
{
    void checkDivisible(int iNum)
    {
        if(iNum % 5 == 0 && iNum % 11 == 0)
        {
            System.out.println("Number is Divisible.");
        }
        else
        {
            System.out.println("Number is not Divisible.");
        }
    }
}

class CP_013
{
    public static void main(String A[])
    {
        Logic lobj = new Logic();

        lobj.checkDivisible(55);
    }
}